test/testdata/workspaces/gradle-sample-app/build.gradle

```diff
@@ -19,7 +22,32 @@ plugins {
 
 repositories {
     // Use Maven Central for resolving dependencies.
-    mavenCentral()
+    maven() {
+        url "${nexus_url}/repository/jcenter/"
+        credentials {
+            username = "${nexus_user}"
+            password = "${nexus_pw}"
+        }
+        allowInsecureProtocol = true
```

Comment:
Maybe we can add a comment here (and in the instances below) that this is only to be set for testing.